Jenkinsfile

```diff
@@ -137,8 +137,12 @@ pipeline {
                                     stage('Deploy latest/version tag') {
                                         when { environment name: 'STREAM', value: 'stable' }
                                         steps {
-                                            sh 'skopeo copy containers-storage:localhost/$IMAGE_NAME docker://docker.io/ucsb/${IMAGE_NAME}:${IMG_PREFIX}latest${IMG_SUFFIX} --dest-username $DOCKER_HUB_CREDS_USR --dest-password $DOCKER_HUB_CREDS_PSW'
-                                            sh 'skopeo copy containers-storage:localhost/$IMAGE_NAME docker://docker.io/ucsb/${IMAGE_NAME}:${IMG_PREFIX}v$(date "+%Y%m%d")${IMG_SUFFIX} --dest-username $DOCKER_HUB_CREDS_USR --dest-password $DOCKER_HUB_CREDS_PSW'
+                                            retry(5) {
```

Comment:
Shouldn't this be `retry(count: 5)` ?

Reply:
I think the int by itself is for the global setting: https://jenkins.io/doc/book/pipeline/syntax/#options
But the official docs seem to indicate the `count` https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#retry-retry-the-body-up-to-n-times

Reply:
It's a distinction without a difference.  The schema has count but the actual pipeline examples in the docs don't include it:

https://www.jenkins.io/doc/pipeline/tour/running-multiple-steps/#timeouts-retries-and-more